### PR TITLE
Warnbutton

### PR DESCRIPTION
**Pull request by thinknoack**
clean up and simplification.

### CODE REVIEW

**Comment by thinknoack** [comment]
gitlab issues #100 #101

File: packages/openneuro-components/src/warn-button/WarnButton.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          size="xsmall"
+        />
+
+        <Button
+          className="btn-warn-component success"
+          iconSize="12px"
+          icon="fa fa-check"
+          color="#fff"
+          backgroundColor="#00b489"
+          onClick={onClick}
```
Comment (david-nishi) [inline packages/openneuro-components/src/warn-button/WarnButton.tsx]: this should also toggle displayOptions when clicked

Comment (david-nishi) [reply packages/openneuro-components/src/warn-button/WarnButton.tsx]: This isn't in the current WarnButton, but i think it'd be good to add a loading state at some point.

**Review by david-nishi — CHANGES_REQUESTED** [review]

**Comment by david-nishi** [comment]
It looks like the ci build is failing. Looks like the signOutAndRedirect prop in UserMenu.tsx needs to be updated to `(history: any) => void` and the Tooltip component directory will need to be made lowercase.

File: packages/openneuro-components/src/warn-button/WarnButton.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const viewAction = (
+    <div className="warn-btn-group " role="group">
+      <div className="slide-in">
+        <Button
+          className="btn-warn-component cancel"
+          iconSize="12px"
+          icon="fa fa-times"
+          color="#fff"
+          backgroundColor="#c00342"
+          onClick={() => setDisplayOptions(!displayOptions)}
```
[the remark this answers is not in the document]
Comment (thinknoack) [reply packages/openneuro-components/src/warn-button/WarnButton.tsx]: the parameter type throws an error. keeping boolean works tho. thoughts?

File: packages/openneuro-components/src/warn-button/WarnButton.tsx (hunk 1 above)
Comment (thinknoack) [reply packages/openneuro-components/src/warn-button/WarnButton.tsx]: created a confirmClick function, let me know if this is a good approach

**Review by david-nishi — APPROVED** [review]

**Pull request merged**